docker/dockerfiles/fe-ubuntu.Dockerfile

```diff
@@ -0,0 +1,28 @@
+# This docker file build the Starrocks fe ubuntu image
+# Please run this command from the git repo root directory to build:
+# DOCKER_BUILDKIT=1 docker build --rm=true -f docker/dockerfiles/fe-ubuntu.Dockerfile -t starrocks/fe-ubuntu:tag .
+
+ARG ARTIFACTIMAGE=artifact:latest
+FROM ${ARTIFACTIMAGE} as artifacts
+
+FROM ubuntu:22.04
+
+RUN apt-get update -y \
+        && apt-get install -y --no-install-recommends default-jdk \
+           curl vim tree net-tools \
```

Comment:
mysql-client is required by fe-entrypoint.sh in order to register itself to FE cluster when the container is started.